docker/flags.go

```diff
@@ -28,6 +28,13 @@ func getHomeDir() string {
 	return os.Getenv("HOME")
 }
 
+func getDaemonConfDir() string {
+	if runtime.GOOS == "windows" {
+		return filepath.Join(os.Getenv("USERPROFILE"), ".docker")
```

Comment:
Is there a standard TODO marker for future windows updates?  I added this mostly as a placeholder and would expect the Windows developers to know what is best.